.github/workflows/wheels.yaml

```diff
@@ -0,0 +1,79 @@
+# Workflow to build and test wheels
+# See [cibuildwheel](https://cibuildwheel.pypa.io/en/stable/) for additional configuration options
+name: Wheel builder
+
+on:
+  # Not running wheel generation on every PR
+  #  pull_request:
+  push:
+    branches:
+      - main
+    tags:
+      - v*
+    workflow_dispatch:
```

Comment:
Do we want to run this on every push to main? I guess it does make sense. Also makes sense for it to be separate from release, nice choice.

I would copy the header style from https://github.com/lewisjared/fgen-example/blob/main/.github/workflows/release.yaml, just for consistency across our files, so it would be something like

```
on:
  # # Comment the line below out for build on PR too i.e. debugging
  # pull_request:
  push:
    branches: ["main"]
    tags: ['v*']
    workflow_dispatch:
```

Reply:
Its a useful smoke test. I've seen some examples where people run a subset of archs on each push to speed things up.